global.json

```diff
@@ -30,7 +30,7 @@
   },
   "msbuild-sdks": {
     "Yarn.MSBuild": "1.15.2",
-    "Microsoft.DotNet.Arcade.Sdk": "5.0.0-beta.20459.8",
-    "Microsoft.DotNet.Helix.Sdk": "5.0.0-beta.20459.8"
+    "Microsoft.DotNet.Arcade.Sdk": "5.0.0-beta.20465.7",
```

Comment:
@dotnet/aspnet-build any reason to not take these changes?

Reply:
Doesn't matter that much as long as any reaction to Arcade changes is also merged forward. We have identical subscriptions for Arcade updates in the two branches.

Reply:
Hmm, this merge doesn't include the work @chcosta did in #25963. We'll now need to duplicate the changes to src/Installers/Windows/SharedFrameworkLib/SharedFrameworkLib.wixproj and src/Installers/Windows/Wix.targets manually in master. Suggest instead including this commit in the merge.